### PR TITLE
add three approvers to CredHub and UAA

### PR DESCRIPTION
**Pull request by swalchemist**
* They are all members of the team at VMware that has been actively maintaining these repos.

### CODE REVIEW

**Comment by Tallicia** [comment]
How can we assign a reviewer to this? @rkoster @bruce-ricard

**Review by bruce-ricard — APPROVED** [review]
They are our team members.

**Review by peterhaochen47 — APPROVED** [review]

**Comment by swalchemist** [comment]
Note that Hongchol and I have been acting as reviewers and approvers under the old structure for more than 3 months, and Alicia for just a few weeks short of 3 months.

**Comment by peterhaochen47** [comment]
Hi @rkoster, could you take a look at this PR please?

**Comment by hsinn0** [comment]
My Justification of asking for the approver role:

As a member of TAS identity and Credential team, I have been maintaining and contributing for UAA and CredHub repo for more than year.  I have been involved in many reviewing/merging PRs, bug fixes and feature implementation.  Approver role is needed for me to continue performing the same kind of tasks.

For UAA, I was author or co-author of 19 commits in develop branch and much more in 74.5.x branch: 
- https://github.com/cloudfoundry/uaa/commits/develop?author=hsinn0
- https://github.com/cloudfoundry/uaa/commits/74.5.x?author=hsinn0

For CredHub, I was author or co-author of more than 20 commits:
- https://github.com/cloudfoundry/credhub/commits/main?author=hsinn0

**Comment by swalchemist** [comment]
I have been contributing to UAA and CredHub since December 2021 with more than 50 commits. We have been making direct commits before we started requiring PRs for CloudFoundry repos. Also, note that many of these commits have been migrated from private LTS repos to the open source repos.

https://github.com/cloudfoundry/uaa/commits/develop?author=swalchemist
https://github.com/cloudfoundry/credhub/commits/main?author=swalchemist

File: toc/working-groups/foundational-infrastructure.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ areas:
     github: peterhaochen47
   - name: Bruce Ricard
     github: bruce-ricard
+  - name: Hongchol Sinn
+    github: hsinn0
+  - name: Danny Faught
+    github: swalchemist
```
Comment (rkoster) [inline toc/working-groups/foundational-infrastructure.md:58]:
```suggestion
    github: swalchemist
  reviewers:
```

File: toc/working-groups/foundational-infrastructure.md [hunk 2]
```diff
@@ -86,6 +92,12 @@ areas:
     github: bruce-ricard
   - name: Markus Strehle
     github: strehle
+  - name: Hongchol Sinn
+    github: hsinn0
+  - name: Danny Faught
+    github: swalchemist
```
Comment (rkoster) [inline toc/working-groups/foundational-infrastructure.md:98]:
```suggestion
    github: swalchemist
  reviewers:
```

**Review by rkoster — CHANGES_REQUESTED** [review]
I tried performing the same analysis for: @Tallicia
- https://github.com/cloudfoundry/uaa/commits/develop?author=Tallicia
- https://github.com/cloudfoundry/uaa/commits/74.5.x?author=Tallicia
- https://github.com/cloudfoundry/credhub/commits/main?author=Tallicia
I could not find any commits. This is fine, however, this would mean that a reviewer role would be more appropriate at this time.

**Review by rkoster — APPROVED** [review]

**Review by beyhan — APPROVED** [review]

**Pull request merged**